src/foscat/xarray/parameters.py

```diff
@@ -0,0 +1,45 @@
+from dataclasses import dataclass, field
+
+import numpy as np
+
+import foscat.scat_cov as sc
+
+
+@dataclass
+class Parameters:
+    """
+    parameters for the scattering covariance transform
+
+    Parameters
+    ----------
+    n_orientations : int
+        The number of orientations of the kernel
+    kernel_size : int
+        The size of the kernel in cells / pixels
+    jmax_delta : float
+        Typically, the scattering covariance coefficients are calculated over N scales,
+        starting from the smallest scale (j=0) up to the largest scale (j_{\text{max}}=\log_2(\text{nside})).
+        Adjusting this parameter eliminates the coefficients at the largest scale
+        where j_{\text{max}}=\log_2(\text{nside})-j_{\text{max\_delta}}.
```

Comment:
it looks like `j` is what we're calling "level" in `xdggs` (other names would be "order" (healpy), "depth" (cdshealpix), "resolution" (h3)). And thus maybe it might be easier to describe this as
```suggestion
        Reduce the maximum level for which scattering covariance coefficients are computed by `delta_level` levels.
        The computed levels are in a range of ``[0, level - delta_level]`` where ``level`` is defined by the input data.
```
Fill in whatever word you prefer for `level`, but if we want to link it to `xdggs` `level` is a natural word for it. Obviously this assumes we rename `jmax_delta` to `delta_level` (but this is your decision). Also, it seems that parameter should be an integer instead, with a default of `0`?